src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -232,6 +234,21 @@ private void addProperties(JsonWriter j, View v)
         }
     }
 
+    // we only do layouts for views with RelativeLayout for now
+    private void addLayouts(JsonWriter j, View v)
+        throws IOException {
+        if (v.getParent() instanceof RelativeLayout) {
```

Comment:
We probably don't need this check- let's just get the layout params and return them, and let the web side decide which can be changed.